docs/quality-assurance/test-strategy.md

```diff
@@ -1,88 +1,103 @@
 # Test Strategy
 
-## Overview
+This strategy details how we add value through testing at the UKHO. Our teams adhere to this strategy when deciding their approach to testing, creating a Test Approach document for their current project.
 
-This strategy details how we add value through testing at the UKHO. The strategy is lightweight and provides a basis for all delivery teams.
+Our core principle is:
 
-## How this document should be used
+> Always use an automation first where possible
 
-Teams should adhere to this strategy when determining their team approach to testing, creating a Test Approach document for the current work-stream that adds detail to this strategy.
+---
 
-## Roles and Responsibilities
+## Methods Of Testing
 
-| Role | Responsibility |
-| - | - |
-| Testers | Champion quality within the team, ensuring quality is present during all stages of software development and inclusive of all team members and users. |
-| Developers | Integrate quality into all stages of development, treating testing as an equal to development |
-| Delivery Managers | Create an environment that fosters quality and integration of all members of a delivery team |
+This section will cover the 3 core methods of testing utilised at the UKHO and key points for each.
 
-## Approach to Testing
+### Automated
 
-### Automation
+Automation should be used where possible.
 
-* Automation should be used where possible. See the [UKHO Test Automation Strategy](test-automation-strategy.md) for more information.
+We suggest that all teams using automation should adhere to these principles:
+
+* New functionality must be covered by passing automated tests (unless there is a legitimate reason not to)
+* Make use of test design patterns and principles
+* Add tests at the correct level of the test pyramid – lower is better
+* All tests should be independent of each other
 
 ### Manual
```

Comment:
Let's rename this to "Manual Scripted" to make an even clearer distinction between this and exploratory